api/WorkerNavigator.json

```diff
@@ -202,10 +202,10 @@
           "spec_url": "https://w3c.github.io/ServiceWorker/#navigator-serviceworker",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
This part of the change looks good, I can't find `serviceWorker` on `WorkerNavigator` in Chromium.